ceph_deploy/osd.py

```diff
@@ -227,6 +228,11 @@ def prepare_disk(
     if journal is not None:
         args.append(journal)
 
+    if block is not None:
+        # The null string is necessary for 'ceph-disk'.
```

Comment:
so this is a requirement from ceph-disk? See this is what my concern is, I had no idea that ceph-disk now requires things like this. Any background information explaining that would be great here

Reply:
@zenglg If you can add the comments before and address the difference, otherwise LGTM